Skills/Perk/PregnancyDesireToBreed.gd

```diff
@@ -0,0 +1,27 @@
+extends PerkBase
+
+func _init():
+	id = Perk.PregnancyDesireToBreed
+	skillGroup = Skill.Pregnancy
+
+func getVisibleName():
+	return "Desire to breed"
+
+func getVisibleDescription():
+	return "Your desire to be bred is so high that new life finds its way inside you with ease. \n" \
+			+ "While not pregnant you deal +30% lust damage and become more susceptible to the lust yourself. \n" \
+			+ "Significantly increases the chances of condoms breaking inside you." #90% to break for normal, quality condoms unaffected
+
+func getSkillTier():
+	return 0
+
+func getPicture():
+	return "res://Images/Perks/Skill_Icon_PregnancyDesireToBreed.png"
+
+#func getBuffs():
+#	if(!npc.menstrualCycle.isVisiblyPregnant()):
+#		return [
+#			buff(Buff.LustDamageBuff, [30]),
+#			buff(Buff.ReceivedLustDamageBuff, [20])
```

Comment:
Ability to toggle perks would be useful here for example.

I think this perk is a bit unbalanced. Its basically an upgrade for characters that can't get pregnant. Although, I do understand that you gotta give births to earn experience for this skill? Still, maybe in the future you will be able to transform your pussy away or something x3

Maybe make it provide buffs only if in heat? Having heat would mean you can get pregnant